search/templates/search/search.html

```diff
@@ -6,19 +6,98 @@
 {% endblock %}
 
 {% block content %}
-<div>
-    <h1>Hey {{ loggedUser.first_name }}!</h1>
-    <h2>Search for apartments now!</h2>
-</div>
-<div> 
-    <form method="POST">
-        {% csrf_token %}
-        <fieldset class="form-group">
-            {{ searchForm|crispy }}
-            <button type="submit">Search!</button>
-        </fieldset>
-    </form>
-    <h8>*This preferences are temporary and can be modified.</h8>
-
+<div class="row">
+    <div class="col-md-5"> 
+        <h2>Hey {{ loggedUser.first_name }}!</h1>
+        <h4>Search for apartments now!</h2>
+        <form method="POST">
+            {% csrf_token %}
+            <fieldset class="form-group">
+                {{ searchForm|crispy }}
+                <button type="submit">Search!</button>
+            </fieldset>
```

Comment:
I recommend you to add some CSS boostrap classes to the button.
Try this:
`<button class="btn btn-primary mt-3" type="submit">{your text}</button>`

`btn btn-primary` stands for the button style.
`mt-3` stands for `margin-top 3px`.

Reply:
I'll try 
@tamirho